python/F3DPythonBindings.cxx

```diff
@@ -133,7 +133,21 @@ PYBIND11_MODULE(pyf3d, module)
     .def("save_buffer", getFileBytes, py::arg("format") = f3d::image::SaveFormat::PNG)
     .def("_repr_png_",
       [&](const f3d::image& img) { return getFileBytes(img, f3d::image::SaveFormat::PNG); })
-    .def("to_terminal_text", [](const f3d::image& img) { return img.toTerminalText(); });
+    .def("to_terminal_text", [](const f3d::image& img) { return img.toTerminalText(); })
+    .def("set_metadata", &f3d::image::setMetadata)
```

Comment:
Can we use def_property here?

Reply:
I don't think so because we're not exposing `metadata` as a dict/map, `get_metadata` and `set_metadata` are still function calls with at least a `key` parameter

Reply:
Understood now :)